tests/pyfunc/test_spark.py

```diff
@@ -22,9 +22,11 @@
 
 
 def score_model_as_udf(model_path, run_id, pandas_df, result_type="double"):
-    spark = pyspark.sql.SparkSession.builder \
-        .config(key="spark.python.worker.reuse", value=True) \
-        .master("local-cluster[2, 1, 1024]") \
+    spark = pyspark.sql.SparkSession.builder\
```

Comment:
Ah good catch - I'm realizing that we intentionally decided to run tests for different model frameworks in different `pytest` invocations in https://github.com/mlflow/mlflow/pull/325/files (IIRC because the memory overhead from importing multiple different model frameworks in the same process could cause Travis tests to OOM). I was going to suggest that we start/stop SparkSessions in each test that uses them, but it seems starting multiple SparkSessions within the same Python process is not well-supported ([link](https://stackoverflow.com/questions/41491972/how-can-i-tear-down-a-sparksession-and-create-a-new-one-within-one-application)).

Thus maybe the right course of action here is to update our contributor guide (in CONTRIBUTING.rst) with test steps that more closely reflect our .travis.yml, as opposed to installing unused mleap dependencies in test_spark.py. I'll file a separate GitHub issue for updating CONTRIBUTING.rst - in this PR I'd suggest we remove changes to test_spark.py